### PR TITLE
added kml_* functions

### PR DESCRIPTION
**Pull request by mkornaukhov03**
kml_* builtins polyfills

### CODE REVIEW

File: kphp_polyfills.php [hunk 1]
```diff
@@ -1467,4 +1467,73 @@ function kphp_tracing_func_enter_branch(int $branch_num) {}
...
 
+#region kml
+
+// builtins for KML format
+
+/**
+ * @param float[][] $features_map_matrix
+ * @return float[]|null
+ */
+function kml_xgboost_predict_matrix(string $model_name, $features_map_matrix) {
```
Comment (Danil42Russia) [inline kphp_polyfills.php:1478]: let's add an `array` typehint :)

File: kphp_polyfills.php [hunk 2]
```diff
@@ -1467,4 +1467,73 @@ function kphp_tracing_func_enter_branch(int $branch_num) {}
...
+ */
+function kml_catboost_predict_vectors(string $model_name, $float_features, $cat_features) {
+  return null;
+}
+
+/**
+ * @param float[] $features_map
+ * @return float|null
+ */
+
```
Comment (unserialize) [inline kphp_polyfills.php:1495]: extra newline

File: kphp_polyfills.php (hunk 1 above)
Comment (mkornaukhov03) [reply kphp_polyfills.php]: Fixed

File: kphp_polyfills.php (hunk 2 above)
Comment (mkornaukhov03) [reply kphp_polyfills.php]: Fixed

**Review by unserialize — APPROVED** [review]

**Pull request merged**